### ISSUE TITLE
3.2.6 Findable Help - overlap with 3.3.3 consistent navigation and other comments

### ISSUE THREAD

**Issue by jpascalides** [w3c/wcag#1394]
A few comments on the Findable Help SC:
1. Would like clarification on what constitutes a human contact mechanism vs. human contact detail.
2. The scope of what "available" constitutes should be defined. Is scope limited to fully functional help mechanisms or is the mere presence of a non-functional human contact mechanism (e.g., a non-functioning phone number to an organization) sufficient? Seems like a loophole that should be closed.
3. Clarify what a "self help" option constitutes in the normative content perhaps by adding the example of "such as an FAQ site"
4. Consider eliminating the blurb of text "in the same relative order" to eliminate overlap and potential source of confusion with 3.2.3 Consistent Navigation.

**Comment by scha14** [comment]
Draft response 

> 1. Examples of human contact details and human contact mechanism are included in the [understanding document](https://www.w3.org/WAI/WCAG22/Understanding/findable-help.html#dfn-web-page). 
> Human contact details such as a phone number, email address, hours of operation.
> Human contact mechanism such as a messaging system, chat client, contact form, social media channel.
> 2. The SC has changed from `available` to `supported` to clarify. The intent of the SC is not to require contact information if a web site is not supported. Does this resolve?
> 3. Examples of self help options are stated in the understanding document in more detail. Since there are several examples for each of the options and it would be inconsistent to only include examples for one, the examples went in the understanding document for readability.
> 4. The same relative order reinforces 3.2.3, except 3.2.3 is for repeating blocks of content. In this case, it is possible to have some help options on one page and not others. The ones that are supported across different pages still appear in the same relative order

**Comment by alastc** [comment]
Hi @jpascalides,
The group met today and agreed this response:
<hr>

1. Examples of human contact details and human contact mechanism are included in the [understanding document](https://www.w3.org/WAI/WCAG22/Understanding/findable-help.html#dfn-web-page). 
Human contact details such as a phone number, email address, hours of operation.
Human contact mechanism such as a messaging system, chat client, contact form, social media channel.

2. The SC has changed from `available` to `supported` to clarify. The intent of the SC is not to require contact information if a web site is not actively supported. Does this resolve?

3. Examples of self help options are stated in the understanding document in more detail. The group has been through an iteration of the SC that included all the examples. However, it is inconsistent with other success criteria, and makes the SC harder to read. Since there are several examples for each of the options and it would be inconsistent to only include examples for one, the examples went in the understanding document for readability.

4. The same relative order reinforces 3.2.3, except 3.2.3 is for repeating blocks of content. In this case, it is possible to have some help options on one page and not others. The ones that are supported across different pages still appear in the same relative order

Please do re-open if you do not consider this issue resolved.
